src/app/app.component.html

```diff
@@ -1,5 +1,5 @@
 <app-header
-  [loggedIn]="isLoggedIn"
+  [loggedIn]="auth.isLoggedIn"
```

Comment:
no, just a minor change in the data flow. Before [logfedIn] input was bound to the `isLoggedIn` property of the `AppComponent`. Now it's bound directly to the Observable of Auth service, which is injected into the AppComponent.